MapboxCoreNavigation/RouteController.swift

```diff
@@ -1064,3 +1107,25 @@ extension RouteController {
         sessionState = SessionState(currentRoute: routeProgress.route, originalRoute: routeProgress.route)
     }
 }
+
+extension RouteController: TunnelIntersectionManagerDelegate {
+    
+    public func tunnelIntersectionManager(_ manager: CLLocationManager,
+                     willEnableAnimationAt location: CLLocation,
+                                           callback: RouteControllerSimulationCompletionBlock?) {
+        tunnelIntersectionManager?.enableTunnelAnimation(for: manager,
+                                             routeController: self,
+                                               routeProgress: routeProgress,
+                                                    callback: callback)
+    }
+    
+    public func tunnelIntersectionManager(_ manager: CLLocationManager,
```

Comment:
These functions need documentation since they are public.

Reply:
@bsudekum Hey, it's already documented in the `TunnelIntersectionManagerDelegate` class: https://github.com/mapbox/mapbox-navigation-ios/pull/1218/files#diff-987c4bbd828e654bd63d59215f890022R20